src/main/java/com/faforever/client/reporting/SupportServiceImpl.java

```diff
@@ -0,0 +1,108 @@
+package com.faforever.client.reporting;
+
+import com.faforever.client.config.ClientProperties;
+import com.faforever.client.fx.PlatformService;
+import com.faforever.client.preferences.PreferencesService;
+import com.faforever.client.update.Version;
+import com.faforever.client.user.UserService;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.io.BufferedReader;
+import java.net.URLEncoder;
+import java.nio.file.Files;
+import java.text.MessageFormat;
+
+@Slf4j
+@Lazy
+@Service
+public class SupportServiceImpl implements SupportService {
+
+  private static final int DEFAULT_READ_LOG_LINES = 100;
+  private static final String MAIL_SUBJECT = "Support request from {0}";
+  private static final String MAIL_TEMPLATE = "Dear Developer Team,\n" +
+      "\n" +
+      "I have encountered the following problem:\n" +
+      "\n" +
+      "**********************************************\n" +
+      "**please tell us what problem you have**\n" +
+      "**********************************************\n" +
+      "\n" +
+      "I would have expected:\n" +
+      "\n" +
+      "*************************************************************\n" +
+      "**please tell what you would have expected instead**\n" +
+      "*************************************************************\n" +
+      "\n" +
+      "Greetings {0}\n" +
+      "\n" +
+      "---------------------------------------------------------------\n" +
+      "username: {0}\n" +
+      "clientVersion: {1}\n" +
+      "supportType: {2}\n" +
+      "system: {3}\n" +
+      "\n" +
+      "error message:\n" +
+      "{4}\n" +
+      "\n" +
+      "lastLogStatements:\n" +
+      "{5}";
```

Comment:
Are we sure that we wanna put this in code instead of a resource file?

Reply:
Yeah we wanna put it in an resource file